src/medCore/gui/toolboxes/medAbstractSelectableToolBox.h

```diff
@@ -35,13 +35,13 @@ class MEDCORE_EXPORT medAbstractSelectableToolBox : public medToolBox
 
     void setSelectorToolBox(medSelectorToolBox *toolbox);
 
+    //! Called when scrolled in medSelectorToolBox
+    virtual void changeSelectedToolBoxEvent(){}
```

Comment:
This function should be removed altogether and ```clear``` used instead, because that's exactly what it does.

As a side note, I know I'm insisting on function names here but maybe it's less obvious to non native english speakers. Wrong function names just make the code unreadable.
Function names are usually "read" as phrases, which means that here we read that this function "changes the selected toolbox event", in the same way that hideEvent is understood to be a function that "hides an event". These are not good names for functions.
This type of naming applies to things like events and signals, which are not read as phrases but as "things that happened".

Example signal/event names:
- clicked
- dragged
- fileLoaded
- destroyed
- fileNotFound

Example slot/function names:
- openWindow
- loadData
- showToolbox

I know sometimes we use things like onClicked for functions, and that is acceptable for virtual functions where you don't know what the behaviour will be, although I think "handleClick" is better.